test/IRGen/Inputs/OtherModule.swift

```diff
@@ -0,0 +1,15 @@
+import resilient_struct
+
+public struct First {}
+public struct Second {
+    public let resilientData: Size
+}
+
+private enum PrivateEnum {
+  case first(First?)
+  case second(Second?)
+}
```

Comment:
I suggest adding an `internal` test as well, since that gets its own case in your switch and will behave differently in the multi-file vs. multi-module tests.